builtin/logical/pki/path_acme_account.go

```diff
@@ -36,6 +36,13 @@ func addFieldsForACMEPath(fields map[string]*framework.FieldSchema, pattern stri
 			Required:    true,
 		}
 	}
+	if strings.Contains(pattern, uuidNameRegex("kid")) {
```

Comment:
I believe this is the wrong fix. On new line 80/old line 73, we attempt to do the same, but we use the wrong regex. I believe updating that regex to use the UUID regex will fix this?

Reply:
Good catch! Thanks for pointing it out :)